css/css-flexbox/flex-basis-011.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Flexbox Test: % flex-basis should not cause engines to treat items as percentage sized</title>
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css-flexbox-1/#flex-basis-property">
+<link href="support/flexbox.css" rel="stylesheet">
+<link rel="match" href="flex-basis-011-ref.html">
+<style>
+.flex-item {
+    flex: 1 0 100%;
+    border: 1px solid blue;
+}
+</style>
+
+<p>Test PASS if there are two boxes with blue borders stretched to fit their contents.</p>
```

Comment:
I'd say "vertically stretched", as that's the part we care in this test.

Reply:
:ok_hand: